data.md

```diff
@@ -234,7 +234,7 @@ You could alternatively calculate `I1 modInt I2`, then add one to the normal int
 ```
 
 The corresponding `<op>Word` operations automatically perform the correct modulus for EVM words.
-
+Operands are assumed to be within the range of a 256 bit EVM word. 
```

Comment:
Formatting (blank line before code blocks). Otherwise line-numbers don't line up between .md and .k files.

Reply:
Also you have a whitespace error at the end of this line.